src/main/java/org/gridsuite/study/server/service/NetworkMapService.java

```diff
@@ -18,8 +18,7 @@
 import org.gridsuite.study.server.dto.InfoTypeParameters;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.core.ParameterizedTypeReference;
-import org.springframework.http.HttpMethod;
-import org.springframework.http.HttpStatus;
+import org.springframework.http.*;
```

Comment:
i am not sure, but i think the team recommend to do explicit import

Reply:
Explicit import s done automatically by the IDE after using 4 imports of the same package but I fixed it manually.